assets/js/base/components/dropdown-selector/index.js

```diff
@@ -31,6 +31,7 @@ import './style.scss';
  * @param {boolean} props.multiple Whether multi-select is allowed.
  * @param {function():any} props.onChange Function to be called when onChange event fires.
  * @param {Array} props.options The option values to show in the select.
+ * @param {boolean} props.isCaseSensitive Whether the dropdown search should be case-sensitive.
```

Comment:
FYI - optional params can be denoted with [] and we can provide default value here as well.

ie. `@param {boolean} [props.isCaseSensitive=false] ...`